src/renderer/containers/Login/LoginContainer.tsx

```diff
@@ -1,28 +1,53 @@
 import * as React from "react";
-import {ReactNode} from "react";
+import {ReactElement, Component} from "react";
 import {Background} from "../../components/Background/Background";
 import {Modal} from "../../components/Modal/Modal";
 import {InputForm} from "../../components/Input/InputForm";
 import {ButtonPrimary, ButtonSecondary} from "../../components/Button/ButtonStandard";
 import {Link} from "react-router-dom";
 import {Routes, OnBoardingRoutes} from "../../constants/routes";
+import {RouteComponentProps} from "react-router";
+import {Notification} from "../../components/Notification/Notification";
+import {Level, Horizontal, Vertical} from "../../components/Notification/NotificationEnums";
+import database from "../../services/db/api/database";
+import {bindActionCreators, Dispatch} from "redux";
+import {connect} from "react-redux";
+import {storeAuthAction,} from "../../actions";
+import {IRootState} from "../../reducers";
 
 interface IState {
     input: string;
+    notification: boolean;
+}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface IOwnProps extends Pick<RouteComponentProps, "history"> {
+}
+interface IInjectedProps{
+    storeAuth: typeof storeAuthAction;
 }
 
-export default class LoginContainer extends React.Component {
+class Login extends Component<
+IOwnProps & IInjectedProps & Pick<IRootState, "register">, IState> {
+
     public state: IState = {
-        input: ""
-    };
-    public handleChange = (e: React.FormEvent<HTMLInputElement>): void => {
-        this.setState({input: e.currentTarget.value});
+        input: "",
+        notification: false,
     };
     
-    public render(): ReactNode {
+    public render(): ReactElement {
         return (
             <Background>
                 <Modal>
+                    <Notification
+                        title="Incorrect password"
+                        isVisible={this.state.notification}
+                        level={Level.ERROR}
+                        horizontalPosition={Horizontal.CENTER}
+                        verticalPosition={Vertical.TOP}
+                        onClose={(): void => {this.setState({notification: false});}}
+                    >
+                    Password does not exist, please register.
```

Comment:
You should have two error messages:
- account doesn't exists
- incorrect password

See comment above